script/e2e-test.sh

```diff
@@ -13,8 +13,11 @@ USE_MULTICLUSTER_OIDC_ENV=${1:-false}
 OLM_VERSION=${2:-"v0.18.2"}
 DEV_TAG=${3:?missing dev tag}
 IMG_MODIFIER=${4:-""}
-DEX_IP=${5:-"172.18.0.2"}
-ADDITIONAL_CLUSTER_IP=${6:-"172.18.0.3"}
+DOCKER_USERNAME=${5:-""}
+DOCKER_PASSWORD=${6:-""}
+TEST_TIMEOUT=${7:-4}
+DEX_IP=${8:-"172.18.0.2"}
+ADDITIONAL_CLUSTER_IP=${9:-"172.18.0.3"}
```

Comment:
Intentionally left these two as last params: not existing when running in GKE. Otherwise params indexes would not be correct.